src/test/kotlin/io/easybreezy/project/application/issue/subscriber/IssueSubscribersTest.kt

```diff
@@ -0,0 +1,107 @@
+package io.easybreezy.project.application.issue.subscriber
+
+import com.google.inject.AbstractModule
+import com.google.inject.Guice
+import com.google.inject.Injector
+import com.google.inject.multibindings.Multibinder
+import io.easybreezy.createIssue
+import io.easybreezy.createMember
+import io.easybreezy.createMyProject
+import io.easybreezy.createProjectRole
+import io.easybreezy.createProjectTeam
+import io.easybreezy.createTeamMember
+import io.easybreezy.infrastructure.event.EventSubscribers
+import io.easybreezy.infrastructure.event.project.issue.Created
+import io.easybreezy.infrastructure.exposed.TransactionManager
+import io.easybreezy.infrastructure.query.QueryExecutor
+import io.easybreezy.project.application.issue.command.language.normalizer.ElementNormalizer
+import io.easybreezy.project.application.issue.command.language.normalizer.LabelNormalizer
+import io.easybreezy.project.application.issue.command.language.normalizer.ParticipantsNormalizer
+import io.easybreezy.project.application.issue.queryobject.IssueParticipantsQO
+import io.easybreezy.project.application.issue.queryobject.IssueQO
+import io.easybreezy.project.application.issue.queryobject.IssueTimingQO
+import io.easybreezy.project.infrastructure.ProjectRepository
+import io.easybreezy.project.model.Repository
+import io.easybreezy.rollbackTransaction
+import io.easybreezy.testDatabase
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Test
+import java.time.LocalDateTime
+import kotlin.test.assertEquals
+
+class IssueSubscribersTest {
+
+    companion object {
```

Comment:
why do you use a companion object? Can it be a property?

Reply:
Because as I knew it can be created several instances for tests running, so I've decided to simplify instances creation